shared/common-adapters/markdown.native.js

```diff
@@ -37,6 +37,8 @@ const codeSnippetBlockTextStyle = {
 
 const quoteBlockStyle = {borderLeftColor: globalColors.lightGrey2, borderLeftWidth: 3, paddingLeft: 8}
 
+// Override styles set by the Markdown component style so they aren't
+// applied to the Text component.
```

Comment:
A suggestion for wording:

```
// The Text component adds default styles which we need to unset so that styles applied to Markdown parent take effect. For instance, we need to unset the default color applied by <Text type="body"> so that <Markdown style={{color: ...}}> works.
```